test-executive-floor.yaml

```
@@ -0,0 +1,370 @@
+
+---
+apiVersion: haconiwa.dev/v1
+kind: Task
+metadata:
+  name: task19_operations_automation
+spec:
+  branch: task19_operations_automation
+  worktree: true
+  assignee: org05-coo-re
+  spaceRef: test-executive-company
+  description: "業務自動化" 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)